contracts/mocks/MockRebasingERC20.sol

```diff
@@ -38,18 +38,8 @@ contract MockRebasingERC20 is ERC20 {
         return true;
     }
 
-    function allowance(address owner, address spender)
-        public
-        view
-        virtual
-        override
-        returns (uint256)
-    {
-        return applyMultiplier(super.allowance(owner, spender));
-    }
-
     function approve(address spender, uint256 amount) public virtual override returns (bool) {
```

Comment:
Since the approve implementation identical to the base class, I dont think you need to override ..

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol#L135